Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG BASEIMAGE=rust:latest
+
+# BUILDER PATTERN
+
+FROM $BASEIMAGE AS builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN apt-get update
+RUN apt-get -y install libclang-dev libmetis-dev libscotch-dev
+
+ARG C_INCLUDE_PATH="/usr/include/x86_64-linux-musl"
+ARG CPLUS_INCLUDE_PATH="/usr/include/x86_64-linux-musl"
+
+RUN rustup target add x86_64-unknown-linux-musl
```

Comment:
Useful only because of `alpine` container below. If we switch to `slim-bullseye` we can remove these lines.